### PR TITLE
RFC: Extending scripts/sign_encrypt.py to verify a signed TA

### PR DESCRIPTION
**Pull request by hoiho-amzn**
Looking for feedbacks on whether to extend `sign_encrypt.py` for
the verification functionality or create a new script.

### CODE REVIEW

File: scripts/sign_encrypt.py [hunk 1]
```diff
@@ -14,6 +14,12 @@
 enc_key_type = {'SHDR_ENC_KEY_DEV_SPECIFIC': 0x0,
                 'SHDR_ENC_KEY_CLASS_WIDE': 0x1}
 
+SHDR_MAGIC = 0x4f545348
+SHDR_SIZE = 20
+
+SHDR_BOOTSTRAP_TA = 1
+SHDR_ENCRYPTED_TA = 2
```
Comment (jforissier) [inline scripts/sign_encrypt.py:21]: Nitpicking... this should be a separate commit (with the changes to lines 201-205)

**Review by jforissier — COMMENTED** [review]
> Looking for feedbacks on whether to extend `sign_encrypt.py` for the verification functionality or create a new script.

No strong opinion, but as far as I'm concerned it is OK to extend the script.

File: scripts/sign_encrypt.py (hunk 1 above)
Comment (hoiho-amzn) [reply scripts/sign_encrypt.py:21]: these enhancements will be on PR https://github.com/OP-TEE/optee_os/pull/5098
